components/nimbus/ios/Nimbus/ArgumentProcessor.swift

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+enum ArgumentProcessor {
+    static func initializeTooling(nimbus: NimbusInterface, args: CliArgs) {
```

Comment:
Is there anything here that needs to be marked `public` or is it fine since it's being used in the `NimbusBuilder`?

Reply:
This is all being accessed from `NimbusBuilder`: so the only public interface is `with(commandLineArgs:)`.